### DISCUSSION TITLE
Exporting comments

### DISCUSSION THREAD

**Discussion by jamesyu** [langfuse/langfuse#4303] [Ideas]
When exporting traces from the UI, is there a way to export WITH the comments on scores? It seems like it only includes the score itself. If not, is there an easy way to do this via code?

**Comment by jannikmaierhoefer** [comment]
Hi @jamesyu, this is currently not possible via the UI - but I agree that this would be helpful. I will turn this post into an "Ideas" post to add this to our roadmap!

In the meantime, you can fetch the comments + scores via the API and merge them with the trace_ids.
